models/mongo/conversations.go

```diff
@@ -0,0 +1,132 @@
+// Package mongo is an implementation of models.AccountsRepository
+// that persists data on a MongoDB database.
+package mongo
+
+import (
+	"accounts-service/models"
+	"context"
+	"errors"
+
+	// "errors"
+
+	"github.com/google/uuid"
+	// "go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+
+	// "go.mongodb.org/mongo-driver/mongo/options"
+	"go.uber.org/zap"
+)
+
```

Comment:
Please configure your IDE so that imports are automatically sorted and added/removed. This will create unecessary diffs and merge conflicts in the future.